### PR TITLE
Create an English translation of "(Nunca) Ouvi Falar de Rust"

### PR DESCRIPTION
**Pull request by ErichDonGubler**
This PR is the culmination of work started at https://github.com/ErichDonGubler/bltavares-presentations/issues/1. Happy to contribute a translation to your awesome Rust presentation, Bruno!

## Remaining work

- [x] Initial translation pass
- [x] Editing pass
    - [x] Handle any TODO/FIXME/XXX comments that Erich can do himself
- [ ] Questions/requests for Bruno
    - [x] Resolution for FIXME/TODO items
    - [x] Code overflows on a screen @erichdongubler tested: https://i.imgur.com/KJB3ZgA.png
        - There are several places this happens. Style problem? :(
    - [x] Some `rustc` errors overflow on display: https://i.imgur.com/m9d9a7S.png
        - There are several places this happens too. :(
    - [x] Localize Nickel walkthrough assets?
    - [x] Anything else you'd like in the translation?

### CODE REVIEW

**Comment by bltavares** [comment]
Hey @ErichDonGubler, thank you so much for the effort translating it. This last weekend I wasn't feelin so well, so sorry for the delay on coming back to it. It means a lot to me that you liked the presentation.

I've generated the HTML for a quick preview of how it is looking like, and I'll go through the TODOs over this weekend. I'll keep you posted.

Meanwhile, I've updated the branch with the HTML. Have a nice day!

**Comment by ErichDonGubler** [comment]
@bltavares: My pleasure! Jamais quero deixar de falar português nem de dar de volta o carinho que tenho recebido do povo brasileiro. :)

Also, just added some more tweaks. It was nice to see the presentation in its entirety -- it let me find a few more QA-ish things I added to the issues in the OP. :)

**Comment by ErichDonGubler** [comment]
@bltavares: My pleasure! Jamais quero deixar de falar português nem de dar de volta o carinho que tenho recebido do povo brasileiro. :) Espero que teja tudo melhorzinho já!

Also, just added some more tweaks. It was nice to see the presentation in its entirety -- it let me find a few more QA-ish things I added to the issues in the OP. :)

**Comment by bltavares** [comment]
@ErichDonGubler Happy new year! Sorry for the delay, the last couple of days were complicated to get back to this (lack of internet, holidays, getting sick and so on).

I've updated the slides with some color and made some small changes. I was thinking of merging it to have it available as a link already, and further changes we make as a new PR. What do you think?

Would you mind checking if the changes I've made read ok in English as well?

Thank you for the work put on it (:

File: nunca-ouvi-falar-de-rust/index.en.html [hunk 1]
```diff
@@ -223,14 +227,14 @@ <h2 id="org26774a6">"Why did they choose to create a low-level language?"</h2>
 </section>
 <section >
 <p>
-Memory management without a garbage collector
+Memory management without a Garbage Collector
```
Comment (ErichDonGubler) [inline nunca-ouvi-falar-de-rust/index.en.html]: Capitalized? Any special reason for this?

File: nunca-ouvi-falar-de-rust/index.en.html [hunk 2]
```diff
@@ -170,13 +175,12 @@ <h2 id="rust"><a id="org53ddb70"></a>Rust</h2>
 
 </section>
 <section >
-
 <p>
-What stood out to me for continued learning:
+What kept me interested on the language:
```
Comment (ErichDonGubler) [inline nunca-ouvi-falar-de-rust/index.en.html]: AH, this is what you meant! Aprende-se algo novo a cada dia. :)

File: nunca-ouvi-falar-de-rust/index.en.org [hunk 3]
```diff
@@ -300,81 +300,81 @@ error[E0282]: type annotations needed
...
+   [[https://play.rust-lang.org/?gist=1bc78bcf4678616aa01538b6a281f9ed&version=stable][Playground]]
 
-    #+BEGIN_SRC rust
-      #[derive(Debug)]
-      struct Request {
-          id: i32
-      }
+   #+BEGIN_SRC rust
+     #[derive(Debug)]
+     struct Order {
```
Comment (ErichDonGubler) [inline nunca-ouvi-falar-de-rust/index.en.org]: Ahhh, também faz sentido isso. Perdão por ter presumido que quis dizer "request"! :)

File: nunca-ouvi-falar-de-rust/index.en.org [hunk 4]
```diff
@@ -455,7 +461,7 @@ $ cargo doc --open
    It's a great evolution for the whole low-level ecosystem
 
    #+reveal: split
-   And a great option to construct low-level platforms
+   And a great option to construct high-level platforms
```
Comment (ErichDonGubler) [inline nunca-ouvi-falar-de-rust/index.en.org]: ...derp.

File: nunca-ouvi-falar-de-rust/index.en.org [hunk 5]
```diff
@@ -485,9 +491,9 @@ $ cargo doc --open
   #+REVEAL_HTML:  <video style="max-width: 80%" controls> <source src="MeuSiteEmRust.webm" type="video/webm"> </video>
 
 ** A website that's:
-   - Light
+   - Lightweight
```
Comment (ErichDonGubler) [inline nunca-ouvi-falar-de-rust/index.en.org]: Oooh, this is a nice mapping to s'more English techno-jargon!

File: nunca-ouvi-falar-de-rust/index.en.org [hunk 6]
```diff
@@ -840,41 +847,47 @@ $ cargo run
...
 
-** And if you want to keep learning about it
+** And if you want to learn more
 
-   - There's a [[https://doc.rust-lang.org/book/][free book]] on Rust's website
-   - [[https://riot.im/app/#/room/#mozilla_#rust-beginners:matrix.org][#rust-beginners]] is a great channel for quickly answering questions
-   - There's [[https://riot.im/app/#/room/#mozilla_#rust-br:matrix.org][#rust-br]] for Portuguese speakers
-   - The [[https://telegram.me/rustlangbr][@rustlangbr]] group on Telegram
+   - There is a [[https://doc.rust-lang.org/book/][free book]] on the website [EN]
+   - [[https://riot.im/app/#/room/#mozilla_#rust-beginners:matrix.org][#rust-beginners]] is a great channel to help with initial doubts
```
Comment (ErichDonGubler) [inline nunca-ouvi-falar-de-rust/index.en.org]: "doubts" is not usually the term native American English speakers use for problems one may run into while learning something -- I'm assuming that what you're saying here is in that vein. I'd definitely use "questions" here.

File: nunca-ouvi-falar-de-rust/index.en.org [hunk 7]
```diff
@@ -840,41 +847,47 @@ $ cargo run
...
+   - There is [[https://riot.im/app/#/room/#mozilla_#rust-br:matrix.org][#rust-br]] for the Portuguese speakers
+   - [[https://telegram.me/rustlangbr][@rustlangbr]] on Telegram
    - [[https://www.meetup.com/topics/rust/][Meetups]]
-   - [[http://exercism.io/languages/rust/about][Exercism]]
-   - And [[https://github.com/rust-br/estudos][online study group]] in Portuguese
+   - [[http://exercism.io/languages/rust/about][Coding exercises]]
+   - And an [[https://github.com/rust-br/estudos][online studygroup]] in Portuguese
```
Comment (ErichDonGubler) [inline nunca-ouvi-falar-de-rust/index.en.org]: "studygroup" isn't a word in English -- is there a reason you removed the space here?

File: nunca-ouvi-falar-de-rust/index.en.org [hunk 8]
```diff
@@ -840,41 +847,47 @@ $ cargo run
...
 
    #+reveal: split
-   If Rust interests you, you can contact me too, because I want to help you with your next steps.
+   If you got interested in Rust, you may get in touch with me as well, I want to help you with your next steps into the ecosystem.
 
    Bruno Tavares - [[https://twitter.com/bltavares][@bltavares]]
 
+** Special thanks
+
+   [[https://github.com/ErichDonGubler][Erich Gubler]]
```
Comment (ErichDonGubler) [inline nunca-ouvi-falar-de-rust/index.en.org]: :D

**Review by ErichDonGubler — COMMENTED** [review]
I'm glad you could clarify some things I wasn't sure about. This is good for me as a translator! I'm glad that this is useful to you. :) I know revision is a lot easier than feeling like you have a montinho of translation work to do...

File: nunca-ouvi-falar-de-rust/index.org [hunk 9]
```diff
@@ -9,6 +9,10 @@
 
   Mas como eu faço um site?
 
+  #+reveal: split
+  There are slides in [[https://presentations.bltavares.com/nunca-ouvi-falar-de-rust/index.en.html%0A][English]]
+  Thanks to [[https://github.com/ErichDonGubler][Erich Gubler]]
```
Comment (ErichDonGubler) [inline nunca-ouvi-falar-de-rust/index.org:14]: :D

**Comment by ErichDonGubler** [comment]
I think that getting this out faster and tweaking later is great! I'd love to see this live. :)

File: nunca-ouvi-falar-de-rust/index.en.html (hunk 1 above)
Comment (bltavares) [reply nunca-ouvi-falar-de-rust/index.en.html]: I'm not sure, I felt like this is a named concept, so we should capitalize it? Would it be better to make it lower cased?

File: nunca-ouvi-falar-de-rust/index.en.org (hunk 3 above)
Comment (bltavares) [reply nunca-ouvi-falar-de-rust/index.en.org]: No problem at all! It is a word with double mapping to English (:

File: nunca-ouvi-falar-de-rust/index.en.org (hunk 6 above)
Comment (bltavares) [reply nunca-ouvi-falar-de-rust/index.en.org]: Thank you, I didn't know about this nuance. I'll change it.

File: nunca-ouvi-falar-de-rust/index.en.org (hunk 7 above)
Comment (bltavares) [reply nunca-ouvi-falar-de-rust/index.en.org]: I'll fix it back hehe

**Comment by bltavares** [comment]
So it has been merged! Thank you for helping with the translation and sorry for how long it took it all. I hope it wasn't too much of a bad experience.

The presentation should be online at some point after Github's and Cloudflare's cache clears up here https://presentations.bltavares.com/nunca-ouvi-falar-de-rust/index.en.html

**Pull request merged**